### DISCUSSION TITLE
Win App

### DISCUSSION THREAD

**Discussion by lemayp** [AppFlowy-IO/AppFlowy#461] [General]
When or how does it work with updating the latest Windows app vs. what has recently been released such as table support?

**Comment by Angel0726** [comment]
> When or how does it work with updating the latest Windows app vs. what has recently been released such as table support?

database

**Comment by annieappflowy** [comment]
> When or how does it work with updating the latest Windows app vs. what has recently been released such as table support?

Hi @lemayp , we are at v0.0.4-beta2 now. v0.0.4 will have the windows application available for you to download and install. The ETA is May 12th.
